api/sql.ts

```diff
@@ -95,9 +95,20 @@ async function updateProf({
 }
 
 /**
- * Initializes mySQL in the backend by creating a connection to the mySQL server
+ * Initializes mySQL in the backend by creating a connection to the mySQL server. See comments within the function for download instructions on mySQL by ctrl (windows) / option (mac) clicking the function name.
  */
 export async function initializeMySQL(): Promise<void> {
+  // Initializing mySQL requires a local mySQL server download from https://dev.mysql.com/doc/refman/8.0/en/installing.html.
+  // Download based on the OS you have and select the recommended developer bundle in the executable.
+  // IMPORTANT NOTE: InnoDB is not required to run mySQL. Simply click next until you can create a username and password.
+  // When running locally, .env should have the following variables:
+  // SQL_PORT=3306
+  // HOST=localhost
```

Comment:
I'd clarify that DB_USER and PASSWORD refers to the user and password that you create, rest looks good